modules/react/card/lib/Card.tsx

```diff
@@ -13,16 +13,38 @@ export interface CardProps extends BoxProps {
    * Children of the Card. Should contain a `<Card.Body>` and an optional `<Card.Heading>`
    */
   children?: React.ReactNode;
+  /**
+   * The variant of the Card. Can be `default`, `borderless` or `filled`.
+   * @default 'default'
+   */
+  variant?: 'default' | 'borderless' | 'filled';
```

Comment:
it's be nice if Design can give you some guidance here. ie "use borderless on colored backgrounds other than white. Use filled when you want a higher emphasis. etc. The same should show in our docs page"